lerna.json

```diff
@@ -0,0 +1,15 @@
+{
+  "packages": [
+    "packages/**",
+    "samples/*"
+  ],
+  "version": "independent",
+  "npmClient": "yarn",
+  "command": {
+    "version": {
+      "ignoreChanges": ["**/*.md/", "**/__tests__/**", "**/*.spec.ts", "**/*.test.ts"],
```

Comment:
this list is never ending, we should rely on semantic messages and common sense instead

Reply:
Doing some testing in https://github.com/haved/cognite-sdk-js

Reply:
then we'll go for a release branch

Reply:
can you create a wildcard? `release/*`?

Reply:
We are no longer using release branch, and even if you want to patch old major versions, you don't use lerna